tests/unittests/core/test_branch_config.py

```diff
@@ -438,11 +429,11 @@ def test_bad_name_experiment(self, parent_config, child_config, create_db_instan
 
         conflict = conflicts.get([ExperimentNameConflict])[0]
 
-        conflict.new_config['name'] = 'should-not-be-overwritten'
+        conflict.old_config['name'] = 'should-not-be-overwritten'
         assert not conflict.is_resolved
         branch_builder.change_experiment_name('test')
         assert len(conflicts.get_resolved()) == 0
-        assert conflict.new_config['name'] == 'should-not-be-overwritten'
+        assert conflict.old_config['name'] == 'should-not-be-overwritten'
```

Comment:
This should be new_config. It's to test that the resolution was reverted properly.